openshift/template.yaml

```diff
@@ -36,6 +36,7 @@ objects:
   data:
     middletier_namespace: ${THOTH_MIDDLEEND_NAMESPACE}
     backend_namespace: ${THOTH_BACKEND_NAMESPACE}
+    storage-bucket-name: ${THOTH_DEPLOYMENT_NAME}
```

Comment:
Personally, I would keep this as deployment-name, the fact that it is also bucket name is transparent now - we will probably use different bucket naming convention in the near future once Sherard comes up with some bucket and prefix naming schema. Here and in the parameter definition below.